src/platform/packages/shared/kbn-scout/README.md

```diff
@@ -154,10 +171,112 @@ test.beforeEach(async ({ pageObjects }) => {
 });
 ```
 
-5. **src/servers/**
+6. **src/servers/**
+
+Here we have logic to start Kibana and Elasticsearch servers using `kbn-test` functionality in Scout flavor. The instance of the `Config` class is passed to start servers for the specific deployment type. The `flags.ts` file contains server-related command-line flags and options. The `loadServersConfig` function not only returns a `kbn-test` compatible config instance, but also converts it to `ScoutServiceConfig` format and saves it on disk to `./scout/servers/local.json` in the Kibana root directory. Scout `config` fixture reads it and expose to UI tests.
+
+### Test Types and Directory Structure
+
+Scout supports two distinct types of tests: UI and API, each with their own directory structure and import patterns:
+
+#### Setting up Test Directory
+
+To get started with Scout testing for your plugin, you need to create the appropriate directory structure in your plugin's root directory:
+
+```
+your-plugin/
+├── test/
+│   └── scout/
+│       ├── ui/
+│       │   ├── playwright.config.ts
+│       │   ├── parallel.playwright.config.ts
+│       │   └── parallel_tests/               # Your UI test specs, that are run in parallel
+│       │   └── tests/                        # Your UI test specs, that are run sequentially
+│       └── api/
+│           ├── playwright.config.ts
+│           └── tests/                        # Your API test specs, that are run sequentially
```

Comment:
Do we also want to mention the `common` folder in this file hierarchy here? Up to you, we could mention that it's optional and it's for shared code like constants and fixtures.

Reply:
good point, we should